lumicks/pylake/force_calibration/convenience.py

```diff
@@ -89,7 +90,7 @@ def calibrate_force(
         measurements performed deep in bulk.
     fast_sensor : bool, optional
          Fast sensor? Fast sensors do not have the diode effect included in the model.
-    sample_rate : float, optional
+    sample_rate : float
```

Comment:
Would it make sense to reorder the parameters in the docstring according to the order in the function header?